modules/any_proxy/any_proxy.go

```diff
@@ -43,8 +43,8 @@ func NewAnyProxy(s *session.Session) *AnyProxy {
 		"Leave empty to intercept any source address."))
 
 	mod.AddParam(session.NewStringParameter("any.proxy.dst_address",
-		session.ParamIfaceAddress,
```

Comment:
hey @BenGardiner can't we just leave the IPv4 default? it's just more convenient for the vast majority of the cases ... what do you think?

Reply:
yup @evilsocket , good call. I think the important change was removing the IPV4 validator and this was collateral damage when I was editing.

Reply:
great! can you update the PR or do you prefer if I do it post merge?

Reply:
I did an online edit to the branch. hopefully that didn't break anything

Reply:
thank you very much! merging 👍🏻